assets/src/helpers/icon.tsx

```diff
@@ -1,132 +1,132 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 // @ts-ignore
-import alertIconSvg from "../../static/images/icon-alert.svg"
+import alertIconSvg from "../../static/images/icon-alert.svg?raw"
```

Comment:
**question:** is the `?raw` a Storybook thing? Have you tested out how this plays along with the prod assets bundle, not just local dev environment?

Reply:
There's a small amount of context in [`0c33423` (#2220)](https://github.com/mbta/skate/pull/2220/commits/0c334236cb4c79589ce2b087286c06c48fa5f6b3)

I'm not sure if this is a storybook thing or a webpack thing, but the `module.rules` that story book uses in it's webpack config has the following defintion
```
  { resourceQuery: /raw/, type: 'asset/source' },
```

This uses https://webpack.js.org/guides/asset-modules/ to import the source instead of `svg-inline-loader`(since it's deprecated and I couldn't get storybook to work with it for some reason).

I've not deployed it yet though, so I'll do so and see if anything is broken.

Reply:
I'll spend some time on this and see if I can get this working without this resource url change, and see if it's useful to get storybook to use our webpack config (or if it even works).

Reply:
SVG's do appear to be working in dev-green after deployment

Reply:
Okay I got it working by adding a rule to use asset-modules for svg's to the storybook webpack config. So I'll revert this commit.

What I'm still not sure of is if we should merge our webpack config with storybook.
We _probably_ should, so that our storybook mimics what the application build would look like in prod, but I'm worried about unseen interactions between our config and storybook's "blessed" config.

It would also require a (AFAIK manual) deep merge of configuration which, isn't great.

Reply:
> What I'm still not sure of is if we should merge our webpack config with storybook.
We probably should, so that our storybook mimics what the application build would look like in prod, but I'm worried about unseen interactions between our config and storybook's "blessed" config.

My gut instinct is that the closer the configs are to each other the better, but I don't understand the Storybook side well enough to have thoughts on that part.

Reply:
If it's compatible, I'd like to convert our webpack.config.js to webpack.config.ts, which would allow us potentially a easier time merging these two configs. (seems like there are some options in storybook that we need to not override for builds to work correctly, like the output file and path)

I'm going to open a ticket to review this in the future instead of gateing this PR on it, since that would involve changes to our webpack config unrelated to storybook.